### PR TITLE
WIP: Standardization/integration

### PR DESCRIPTION
**Pull request by AxelPeter**
Related to https://github.com/ovh-ux/ovh-manager-web/pull/258

### CODE REVIEW

**Comment by antleblanc** [comment]
PR to `develop` instead?

**Comment by AxelPeter** [comment]
Yes, changed to develop :)

File: src/microsoft/office/license/edit/password/microsoft-office-edit-password.html [hunk 1]
```diff
@@ -88,20 +88,11 @@
...
-                                    <span class="oui-checkbox__icon">
-                                        <span class="oui-icon oui-icon-checkbox-unchecked" aria-hidden="true"></span>
-                                        <span class="oui-icon oui-icon-checkbox-checked" aria-hidden="true"></span>
-                                        <span class="oui-icon oui-icon-checkbox-checkmark" aria-hidden="true"></span>
-                                    </span>
-                                </label>
-                            </div>
+                            <oui-checkbox id="email-admin" name="email-admin"
+                                disabled="!editPwdCtrl.model.password && !editPwdCtrl.password.confirm"
+                                model="editPwdCtrl.model.shouldSendMail"
```
Comment (FredericEspiau) [inline src/microsoft/office/license/edit/password/microsoft-office-edit-password.html]: `data-` for non HTML5 attributes

**Comment by AxelPeter** [comment]
The navbar will be added later. 
First, upgrade `ovh-ui-kit` and `ovh-ui-angular` to v2.0.0.

**Pull request closed without merging**